graphics/src/Image.cc

```diff
@@ -242,6 +242,28 @@ void Image::SetFromData(const unsigned char *_data,
   }
 }
 
+//////////////////////////////////////////////////
+void Image::SetFromCompressedData(const unsigned char *_data,
+                                  unsigned int _size,
+                                  Image::PixelFormatType _format)
+{
+  if (this->dataPtr->bitmap)
+    FreeImage_Unload(this->dataPtr->bitmap);
+  this->dataPtr->bitmap = nullptr;
+
+  if (_format == COMPRESSED_PNG)
+  {
+    FIMEMORY *fiMem = FreeImage_OpenMemory(const_cast<unsigned char*>(_data), _size);
```

Comment:
`const_cast` is a little concerning here.  Can we change the API to reflect that `const`-ness may be violated?

Reply:
Done https://github.com/gazebosim/gz-common/pull/368/commits/8054b0d78433da3de936ffc9b81d87295207a26f